docs/en_US/jailbreak/installing-palera1n-tvos-15.0.md

```diff
@@ -0,0 +1,250 @@
+1. Run `palera1n -f`.
+1. When ready, press `Enter` and follow the on-screen instructions to enter <router-link to="/faq/#what-is-dfu-mode">DFU mode</router-link>.
+
+You'll need to do this every time you rejailbreak your device.
+:::
+
+::: tip
+
+A9(X) and earlier devices have an issue where they will get stuck midway through this process in pongoOS. To work around this, you'll need to do the following:
+
+1. In the terminal window, press `Control` + `C` on your keyboard.
+1. Rerun the command that you just ran.
+
+:::
+
+Once the device boots up, open the palera1n loader app and tap `Install`. After some time, you'll be prompted to set a passcode for using command-line tools, and then `purePKG` should be on your home screen.
+
+::: tip
+
+To rejailbreak your device, simply run the last command again and then repeat any other applicable steps.
+
+:::
+
+::::
+
+:::: tab name="Linux"
+
+
+::: danger
+
+If you are trying to use Virtual Machine software from Windows (e.g., VirtualBox, VMWare, Windows Subsystem for Linux, etc.), you will not succeed with this guide.
+
+:::
+
+::: warning
+
+If you are using a computer with an AMD Ryzen CPU, you will likely run into issues. If you do, you should use a Mac or a computer with an Intel CPU to follow this guide.
+
+:::
+
+### Installing palera1n
+
+1. Open a terminal window.
+1. Run `sudo systemctl stop usbmuxd`.
+1. Run `sudo usbmuxd -f -p`.
+1. Open another terminal window.
+1. Run `sudo /bin/sh -c "$(curl -fsSL https://static.palera.in/scripts/install.sh)"`.
+
+
+### Running palera1n
+
+::: details Apple TV 4K instructions
+1. Plug your GoldenEye/Foxlink cable into the Ethernet port on your Apple TV.
+1. Plug your DCSD cable into the Lightning connector on the GoldenEye/Foxlink cable upside down-the side labeled "this side up" or similar should face down; otherwise, it might not enter <router-link to="/faq/#what-is-dfu-mode">DFU mode</router-link>.
+1. Finally, plug the USB end of your DCSD cable into your computer or any power source.
+1. Restart your Apple TV with both cables plugged in to put it into DFU mode.
+1. Once in DFU mode, unplug your DCSD cable (**not** your GoldenEye/Foxlink cable, keep that plugged in) and plug in your Lightning cable, with the other end connected to your computer.
+1. Run `palera1n -f`.
+
+You'll need to do this every time you rejailbreak your device.
+:::
+
+::: details Apple TV HD instructions
+1. Plug your USB-C cable into the USB-C diagnostic port on the back of the device and the other end into your computer.
+1. Run `palera1n -f`.
+1. When ready, press `Enter` and follow the on-screen instructions to enter <router-link to="/faq/#what-is-dfu-mode">DFU mode</router-link>.
+
+You'll need to do this every time you rejailbreak your device.
+:::
+
+::: tip
+
+A9(X) and earlier devices have an issue where they will get stuck midway through this process in pongoOS. To work around this, you'll need to do the following:
```

Comment:
Does this apply to Apple TV's? If it does, the start of this sentence should be changed to "The Apple TV HD has an issue".